code/game/objects/items/weapons/manuals.dm

```diff
@@ -499,6 +499,165 @@
+				</head>
+				<body style="font-size: 18px;color:#F7D61A;background-color:#221111;">
+
+				<h1><img src='data:image/png;base64,[icon2base64(icon('icons/logos.dmi', "nano-logo"))]' style='position: relative; top: 6;'>Enemies of Nanotrasen: A Quick Overview<img src='data:image/png;base64,[icon2base64(icon('icons/logos.dmi', "nano-logo"))]' style='position: relative; top: 6;'></h1>
+                <h2>Foreword</h2>
+				<p>A Mega-Corporation the size of Nanotrasen doesn't carve itself a piece of the intergalactic cake without making itself a few enemies along the way. Identifying agents of those enemy factions and thwarting their plans falls into the job description of our Security forces, but every Nanotrasen employee is encouraged to report any suspicious activities, and should the situation call for it, take up arms against them. These antagonistic forces are dangerous and often use underhanded tactics, this guide aims to even the ground even slightly by giving you some information on how they operate, and how they can be dealt with.
+
+
+				<h2>Chapters</h2>
+
+				<ol>
+					<li><a href="#Syndicate">The Syndicate</a></li>
+					<li><a href="#Vampire">The Vampire Lords</a></li>
+					<li><a href="#Changeling">The Changeling Hivemind</a></li>
+					<li><a href="#Wizard">The Wizard Federation</a></li>
+					<li><a href="#Cult">The Cult of Nar-Sie</a></li>
+					<li><a href="#Revolution">The Revolutionaries</a></li>
+					<li><a href="#Blob">The Blob Conglomerate</a></li>
+					<li><a href="#Malf">The Free Silicon Hivemind</a></li>
+					<li><a href="#Vox">The Vox Shoal</a></li>
+					<li><a href="#Ninja">The Spider Clan</a></li>
+					<li><a href="#Minor">other minor factions and threats of note</a>
+                    <ol>
+					<li><a href="#Alien">The Alien Hivemind</a></li>
+					<li><a href="#Catbeast">Feral Catbeasts</a></li>
+					<li><a href="#Time">The Time Agency</a></li>
+					<li><a href="#Rambler">Soul Ramblers</a></li>
+					<li><a href="#Plague">Plague Mice</a></li>
+					<li><a href="#Grue">Grues</a></li>
+					<li><a href="#Pulse">Pulse Demons</a></li>
+					<li><a href="#Spider">Giant Space Spiders</a></li>
+                    </ol>
+                    </li>
+				</ol>
+
+
+				<h2><img src='data:image/png;base64,[icon2base64(icon('icons/logos.dmi', "synd-logo"))]' style='position: relative; top: 8;'><a id="Syndicate">The Syndicate</h2>
+
+                <p>The Syndicate are a collaboration of enemies to Nanotrasen. They are the most notorious threat to our space stations as they have made it their official goal to challenge Nanotrasen's monopoly on the sector. They are known to commit every possible crime from thievery, murder, sabotage, and the outright destruction of stations via nuclear devices. Have no qualms about killing them, as they will not hesitate to kill you unless it directly hinders their mission. They are known to work in red, due to the reuse of old soviet spacesuits from the fourth cold war era.
```

Comment:
```suggestion
                <p>The Syndicate is an ill-defined alliance of all enemies to Nanotrasen. They are the most notorious threat to our space stations as they have made it their official goal to challenge Nanotrasen's influence on the sector. No crime is below a Syndicate agent. Thievery, murder, sabotage, and the outright destruction of stations via nuclear devices are among their observed tactics. These enemies of the corporation are to be neutralised, detained, or in extreme cases, terminated on sight. Their red attire has been attributed to their scavenging of old soviet spacesuits from the fourth Cold War era.
```
Didn't like the open encouragement of validating on sight.

Reply:
I wouldn't call the Syndicate an "ill-defined" alliance. Since they are the threat that Nanotrasen has to deal with the most, we can expect them to know the most about them in turn. Ok for the rest.